### ISSUE TITLE
A new need: setting the Info of swagger file

### ISSUE THREAD

**Issue by CunliangGeng** [CLARIAH/grlc#289]
The current code does not allow to set the info (highlighted text in the figure below. Also see the [Info object](http://spec.openapis.org/oas/v3.0.3#info-object) in OpenAPI spec). I expect it possible to e.g. change the text `local` to `Local GRLC Server`, add some introduction text about the server under `local`, ad contact info, link Apache 2.0 to the `license`, etc.  

![image](https://user-images.githubusercontent.com/9798985/93606983-07091580-f9c9-11ea-8719-57858c5efc3e.png)

**Comment by c-martinez** [comment]
It is not too difficult to configure some of these information. I think instead of adding this information to the `config.ini` it might be better to include this information in a configuration file stored in the `local_sparql_dir`. This config file would have the following format:

```ini
[repo_info]
repo_title = Awesome repo
contact_name = Contact name
contact_url = https://mypage.org
licence_url = https://apache.org
```

Would that fit what you need @CunliangGeng ?

**Comment by CunliangGeng** [comment]
Thanks, it looks great! I'd also have the description of the server, and it'd be great to also add `repo_description = A very awesome repo ...` :-)

**Comment by c-martinez** [comment]
Good idea -- the swagger definition does not currently include a description, but it is probably a good idea to include one :-)

**Comment by c-martinez** [comment]
Fixed in v1.3.5
